### ISSUE TITLE
readme_en and readme_de versions details not applied

### ISSUE THREAD

**Issue by stronk7** [ApacheFriends/xampp-build#12]
Hi,

first of all, thanks for Xampp, great work!

Secondly, I'm not sure if this is the correct place to report this, so you're welcome to point me to another place.

I've just downloaded `xampp-portable-windows-x64-8.0.25-0-VS16.zip` and, looking to the `readme_en`and `readme_de` files, they are missing any versions information and, instead, they are showing some placeholders.

```
###### ApacheFriends XAMPP Version 8.0.25 ######
 
Note: The Portable Version does not contain the FileZilla FTP and the Mercury Mail Server. The service installations are also disabled here.


Important! PHP in this package needs the Microsoft Visual C++ @@BITROCK_VCREDIST_VERSION@@ Redistributable package from
http://www.microsoft.com/en-us/download/. Please ensure that the VC++ @@BITROCK_VCREDIST_VERSION@@ runtime
libraries are installed on your system.

  + Apache @@BITROCK_APACHE_VERSION@@
  + MariaDB @@BITROCK_MARIADB_VERSION@@
  + PHP @@BITROCK_PHP_VERSION@@ + PEAR
  + phpMyAdmin @@BITROCK_PHPMYADMIN_VERSION@@
```

The previous version that I was using (`xampp-portable-windows-x64-7.4.28-1-VC15.zip`) has the placeholders properly replaced by real versions.

Ciao :-)

**Comment by beltran-rubo** [comment]
Thanks! That is the perfect place to report it, we will look into it for the next release.

**Comment by beltran-rubo** [comment]
Fixed at the 8.2.0 version and upcoming releases.
